test/sky-pages-module-generator.spec.js

```diff
@@ -1,24 +1,73 @@
 /*jshint jasmine: true, node: true */
 'use strict';
 
-const codegen = require('../utils/codegen-utils');
+const mock = require('mock-require');
```

Comment:
This spec file was affecting the state of other unit tests, so I mocked out its dependencies and made sure those dependencies' unit tests were covered 100%.